samples/Exporters/Console/TestApplicationInsights.cs

```diff
@@ -32,27 +32,25 @@ internal static object Run()
             DistributedContext.Carrier = AsyncLocalDistributedContextCarrier.Instance; // Enable asynclocal carrier for the context
             DistributedContext dc = DistributedContextBuilder.CreateContext(FrontendKey, "mobile-ios9.3.5");
 
-            using (var tracerFactory = TracerFactory.Create(builder => builder
+            using var tracerFactory = TracerFactory.Create(builder => builder
                 .SetResource(Resources.CreateServiceResource("my-service"))
-                .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key")))
-            {
-                var tracer = tracerFactory.GetTracer("application-insights-test");
+                .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key"));
+            var tracer = tracerFactory.GetTracer("application-insights-test");
 
-                using (DistributedContext.SetCurrent(dc))
-                using (tracer.StartActiveSpan("incoming request", out var span))
-                {
-                    span.AddEvent("Start processing video.");
-                    Thread.Sleep(TimeSpan.FromMilliseconds(10));
-                    span.AddEvent("Finished processing video.");
-                }
+            using (DistributedContext.SetCurrent(dc))
+            using (tracer.StartActiveSpan("incoming request", out var span))
+            {
```

Comment:
Right, not here because there are stil scopes `{ }` so it didn't use the feature on all Disposables.
It is quite hard to review this since it's mostly identation

Reply:
Yea, exactly. Github is not helpful with kinda `scope` changes.